lua/wire/server/wirelib.lua

```diff
@@ -506,8 +505,8 @@ function WireLib.TriggerOutput(ent, oname, value, iter)
 
 	local output = ent.Outputs[oname]
 	if (output) and (value ~= output.Value or output.Type == "ARRAY" or output.Type == "TABLE") then
-		if (output.TriggerLimit <= 0) then return end
```

Comment:
This return is supposed to block too many triggers, but you removed it and didn't put in place a different check. How is the triggering limited?